xml/System.Security.Cryptography.X509Certificates/X500DistinguishedNameBuilder.xml

```diff
@@ -14,8 +14,26 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>This class facilitates building a distinguished name for an X.509 certificate.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When constructing the SEQUENCE OF Relative Distinguished Names, this builder constructs the SEQUENCE OF in the opposite order which they were added to the builder. For example:
```

Comment:
Should SEQUENCE OF be code-fenced so it's not localized?

Reply:
It's referencing the ASN.1 data type, not something within .NET, so it'd just be the monospace style ("When constructing the `SEQUENCE OF` Relative Distinguished Names, ...").

Not sure how well "Relative Distinguished Names" localizes, either, since that's a datatype/concept from the ITU X.500 specification.

I defer to your expertise on what to do there.

Reply:
I'll fence it, @gewarren:

```suggestion
When constructing the `SEQUENCE OF` `Relative Distinguished Names`, this builder constructs the` SEQUENCE OF` in the opposite order which they were added to the builder. For example:
```